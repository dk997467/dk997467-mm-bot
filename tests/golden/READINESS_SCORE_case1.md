READINESS SCORE

Result: GO  Score: 90.000000

| section | score |
|---------|-------|
| edge | 30.000000 |
| latency | 25.000000 |
| taker | 15.000000 |
| guards | 10.000000 |
| chaos | 10.000000 |
| tests | 10.000000 |


