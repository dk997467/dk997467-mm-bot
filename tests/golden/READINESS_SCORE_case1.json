{"runtime":{"utc":"1970-01-01T00:00:00Z","version":"0.1.0"},"score":100.0,"sections":{"chaos":10.0,"edge":30.0,"guards":10.0,"latency":25.0,"taker":15.0,"tests":10.0},"verdict":"GO"}
